lib/nerves_network/application.ex

```diff
@@ -7,11 +7,19 @@ defmodule Nerves.Network.Application do
     import Supervisor.Spec, warn: false
 
     [resolvconf_file: resolvconf_file] = Application.get_env(:nerves_network, :resolver, [])
+    [ipv4: 
+     [
+       lease_file: _lf,
+       pid_file: _pf,
+       config_file: dhclientv4_config_file
+     ]
+    ] = Application.get_env(:nerves_network, :dhclientv4, [])
```

Comment:
@metelik  This still needs to be fixed.  You can't pattern match like this

Reply:
I fixed it :/